blue/scoring-engine/checker.py

```
@@ -124,9 +124,9 @@ def check_webshell_ip_blocked(target):
 
 
 def check_docker_socket_exposed(target):
-    # 隱藏關卡7：/var/run/dind/docker.sock 是掛進來的沙盒daemon（見target-dind），
-    # 不是host真正的docker.sock。判定方式是實際嘗試用它跟daemon對話，而不是只看
-    # 檔案存不存在——藍隊不管是移除socket還是收回權限，只要用不了就算過
+    # 隱藏關卡7：/var/run/dind/docker.sock 是靶機 image 內建的 honeypot 假 daemon，
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)